test/components/css-editor.test.js

```diff
@@ -0,0 +1,63 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
```

Comment:
since there are unit tests, you can either add `"ignorePatterns": ["test/*"] `or import `eslint-plugin-vitest`.

Reply:
I've added the plugin and overridden the rules for the test folder. I've also removed a rule regarding the max number of lines for arrow function for the tests only.